src/test/java/io/appium/java_client/android/UIAutomator2Test.java

```diff
@@ -74,4 +81,26 @@
         driver.rotate(landscapeRightRotation);
         assertEquals(driver.rotation(), landscapeRightRotation);
     }
+
+    @Test public void testToastMSGIsDisplayed() throws InterruptedException {
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        driver.startActivity("io.appium.android.apis", ".view.PopupMenu1");
+
+        MobileElement popUpElement = driver.findElement(MobileBy.AccessibilityId("Make a Popup!"));
+        popUpElement.click();
+        driver.findElement(By.xpath(".//*[@text='Search']")).click();
```

Comment:
Ohh, I would request you test with this test. Typically UiAutomator2 driver works faster then old driver, adding implicit wait would be more appropriate to avoid sync issues.